### PR TITLE
Emoji Endpoints

### PR DESCRIPTION
**Pull request by DedShotTM**
This pull request will add the emoji endpoints.

### CODE REVIEW

File: src/rest/Endpoints.ts [hunk 1]
```diff
@@ -269,4 +269,22 @@ export default class Endpoints {
...
     return `/guilds/${guildID}/vanity-url`;
   }
+
+  /**
+   * `/guilds/${guildID}/emojis/${emojiID}`
+   * - GET
+   * - PATCH
+   * - DELETE
+   */
+  public static EMOJI(guildID: string, emojiID: string) {
```
Comment (Khaaz) [inline src/rest/Endpoints.ts:279]: This should probably named `GUILD_EMOJI`
```suggestion
  public static GUILD_EMOJI(guildID: string, emojiID: string) {
```

File: src/rest/Endpoints.ts [hunk 2]
```diff
@@ -269,4 +269,22 @@ export default class Endpoints {
...
+    return `/guilds/${guildID}/emojis/${emojiID}`;
+  }
+
+  /**
+   * `/guilds/${guildID}/emojis`
+   * - POST
+   */
+  public static CREATE_EMOJI(guildID: string) {
```
Comment (Khaaz) [inline src/rest/Endpoints.ts:287]: This also support GET.
Should be named GUILD_EMOJIS as well for consistency
```suggestion
  public static GUILD_EMOJIS(guildID: string) {
```

**Review by Khaaz — CHANGES_REQUESTED** [review]

File: src/rest/Endpoints.ts (hunk 2 above)
Comment (DedShotTM) [reply src/rest/Endpoints.ts]: Done

File: src/rest/Endpoints.ts (hunk 1 above)
Comment (DedShotTM) [reply src/rest/Endpoints.ts]: Done

**Review by Khaaz — APPROVED** [review]

**Pull request merged**